myapp/templates/single_stock.html

```diff
@@ -2,10 +2,24 @@
 
 {% block content %}
 <div class="col-md-12 text-left">
-		<h2 class="display-4">
-				{{ data.companyName }}
-				<small class="text-muted">({{ data.symbol }})</small>
-		</h2>
+	<table>
+		<tr>
+			<td><h2>{{ data.companyName }}<small class="text-muted">({{ data.symbol }}) &emsp14;</small></h2></td>
+			{% if request.user.is_authenticated %}
+			<td>{% if stock in profile.watchlist.all %}
+				<form method="post" action="wremove"> {% csrf_token %}

```

Comment:
use this, to be more centralized
```suggestion
				<form method="post" action={% url 'watchlist_remove' %}> {% csrf_token %}

```

Reply:
Fixed!